src/burnchains/burnchain.rs

```diff
@@ -321,10 +340,20 @@ impl Burnchain {
             consensus_hash_lifetime: params.consensus_hash_lifetime,
             stable_confirmations: params.stable_confirmations,
             first_block_height: params.first_block_height,
-            first_block_hash: params.first_block_hash.clone()
+            first_block_hash: params.first_block_hash.clone(),
+            reward_cycle_period: 1000,
         })
     }
 
+    pub fn is_reward_cycle_start(&self, block_height: u64) -> bool {
```

Comment:
There's a Clarity function in the PoX contract that does this.  Should there be a single source of truth for determining (1) what reward cycle we're in and (2) where in it we currently are?

Reply:
I think it's important for this to be invoked without instantiating a Clarity VM or full chainstate -- if we want to make sure that there's a single source of truth for this, then the Clarity contract should not be computing this.

Reply:
Sounds good.  I can expose this function (and others) to the Clarity VM then, if needed.